content/news/2024-12-10-ms8-workshop/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Evaluating Tools and Services to improve the Quality of Research Software"
+draft: false
+date: 2024-12-10T00:00:00Z
+start_date: 2024-12-10T00:00:00Z
+end_date: 2024-12-10T00:00:00Z
+summary: >
+    Work Package 3 is launching an interactive workshop to examine which tools and services researchers use to improve research software quality.
+
+---
+
+[Work package 3](everse.software/workpackages/03_tools_and_services/) - _Tools and Services_ is organising an interactive workshop to examine which tools and services researchers use to improve the quality of research software, and how they use them!
+
+**<font color="40A3DC">Save the date: Monday 10th February 2025, 10:00-16:00 CET.** </font> \
+Please register your interest [here](https://forms.office.com/Pages/ResponsePage.aspx?id=sAafLmkWiUWHiRCgaTTcYY8mG3jqdkZDlfFoAQVJfklUNVNQMEVQUDQ4SUg1MVpXRzRIMDZONDYwQi4u). 
+
+
+**Is this workshop for you? Join us if you:**
+* Work with/develop research software.
+* Work with or want to develop software of high quality (considering FAIRness, sustainability, portability, testing, security, etc.).
```

Comment:
these two lines read redundant 
I'd rephrase:
- Work with or develop research software
- Want to improve your software quality (considering FAIRness, sustainability, portability, testing, security, etc.).

Reply:
Yes I agree - I tried to make it clearer between working with software and striving to improve it, along the lines of what you suggested